soh/soh/Enhancements/randomizer/randomizer_check_objects.h

```diff
@@ -25,6 +25,10 @@ typedef enum {
     RCTYPE_CHEST_GAME, //todo replace this once we implement it, just using it to exclude for now
     RCTYPE_LINKS_POCKET, //todo this feels hacky
     RCTYPE_GOSSIP_STONE,
+    RCTYPE_SONG_LOC, // Song locations
```

Comment:
i see you changed this from `SONG_LOCATION` to `SONG_LOC`, I prefer the more verbose `SONG_LOCATION`